README.md

```diff
@@ -107,6 +107,13 @@ Note that for all functions inside `nx_code.py` that do not have an nx-parallel
 import networkx as nx
 import nx_parallel as nxp
 
+# enabling networkx's config for nx-parallel
```

Comment:
I'm wonder if putting config issues into this section is a ds=istraction. They are trying to understand backend usage, not how to configure. Maybe a comment line pointing to the config docs here is sufficient. If you want to keep this here, there should at least be a word indicating that the config stuff is optional. 
```suggestion
# optional enabling networkx's config for nx-parallel
```

Reply:
I don't think it is optional. We don't get any errors when we run with the default configs. But, we are also not running the algorithms on multiple cpu cores or threads because `n_jobs=None` by default. So, to actually run an algorithm as a parallel algorithm they would have to set atleast the `n_jobs` config to something non 0 or non None. I have updated the README.md accordingly.

Reply:
Wait -- the default value gives no parallel computation?  That doesn't seem right to me. Shouldn't the default be to use all the cpus we can get?

Reply:
> Wait -- the default value gives no parallel computation? That doesn't seem right to me.

I understand your concern. However, this behavior is consistent with `joblib.parallel_config`, where parallel computation isn’t enabled by default (i.e. `n_jobs=None` by default). Since modifying the defaults within `joblib.parallel_config` is outside my control(bcoz that would mean changing joblib's codebase), changing the default `n_jobs` in NetworkX’s configuration from `None` to `-1` will introduce inconsistencies between the two configuration systems, and increase differences between the two config systems, leading to user confusion and potential pitfalls.

> Shouldn't the default be to use all the cpus we can get?

That was indeed the previous default, but I've since updated the documentation to reflect the current behavior. Reverting this change would necessitate another extensive documentation update across the entire package, which might not be feasible before the GSoC deadline.

Reply:
That sounds great